api/v1beta2/ibmpowervsmachine_types.go

```diff
@@ -77,13 +77,12 @@ type IBMPowerVSMachineSpec struct {
 
 	// systemType is the System type used to host the instance.
 	// systemType determines the number of cores and memory that is available.
-	// Few of the supported SystemTypes are s922,e880,e980.
-	// e880 systemType available only in Dallas Datacenters.
+	// Few of the supported SystemTypes are s922,e980,s1022,e1050,e1080.
 	// e980 systemType available in Datacenters except Dallas and Washington.
```

Comment:
```suggestion

```

I see e980 available in all washington and dallas datacenters (except only dal14)
ref - https://cloud.ibm.com/power/overview

as @Karthik-K-N said we can validate this via webhook.

Reply:
I think we can merge this PR as of now

But to validate it in webhook, I think that wont be easy because, Currently in machine spec we set system type and service instance either by name/id. Without much computation we can't find to which zone this service instance belong. We have zone info set only in IBMPowerVSCluster spec.

Given that we are doing some spec changes to accommodate multi workspace we can handle this scenario later.